com/win32com/client/__init__.py

```diff
@@ -490,6 +493,20 @@ def Record(name, object):
 # The base of all makepy generated classes
 ############################################
 class DispatchBaseClass:
+
+    # _prop_map_*_ are set in generated subclasses by gen_py
+    _prop_map_get_: ClassVar[
+        dict[
+            str,
+            tuple[
+                int, int, tuple[int, int], tuple[tuple[int, int], ...], str, str | None
+            ],
+        ]
+    ]
+    _prop_map_put_: ClassVar[
+        dict[str, tuple[tuple[int, int, int, int], tuple[int, ...]]]
```

Comment:
I am not certain about `tuple[int, ...]` since I couldn't find a clear example of what to pass to `Invoke` past the 5th argument.